### PR TITLE
append Aixt project

### PR DESCRIPTION
**Pull request by fermarsan**
Append a beta project named "Aixt" that implements a subset of V syntax for microcontrollers. 
[https://github.com/fermarsan/aixt](https://github.com/fermarsan/aixt)

### CODE REVIEW

**Comment by JalonSolov** [comment]
(case-insensitive) Alpha order, please.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 - [Applications](#applications)
 	- [Command-line](#command-line)
+	- [Aixt](https://github.com/fermarsan/aixt)
```
Comment (JalonSolov) [inline README.md]: This doesn't look like the correct place.  It should go down in the command line section, not in the TOC.

**Comment by fermarsan** [comment]
I will correct it soon, thanks

**Comment by Larpon** [comment]
Why was this closed? All checks has passed?

**Comment by Delta456** [comment]
> Why was this closed? All checks has passed?

Because another PR was made with the correct and that got merged https://github.com/vlang/awesome-v/pull/216

**Comment by Larpon** [comment]
Right, all good 👍🏻

**Pull request closed without merging**